### DISCUSSION TITLE
Despliegue de página web

### DISCUSSION THREAD

**Discussion by ArielCanedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#255] [Ideas]
Compañeros explorers.
Les quiero comprar una forma de desplegar su página web desde GitHub
Cuando suben su repositorio con archivos HTML se van a ajustes en la sección de "Code and automation" está la pestaña de "pages"
Se meten allí y pueden desplegar su página en internet
Si su repo tiene el archivo readme
Pueden crear un hipervinculo a los archivos HTML
Algo así
< a href="pastelcliente.html">Enlace página para el cliente < /a> (quitando los espacios después de "<") 
El archivo readme funciona parecido a un HTML donde pueden editarlo y los archivos html que suban los tomará de allí mismo
así como en el ejemplo que puse "tuarchivos.html" y ya aparecerá un link para entrar
Espero que les sirva de ayuda y si tienen dudas aquí estaré

**Comment by JuanSalvador73** [comment]
Muchísimas gracias por el tip :D

**Reply by ArielCanedo** [reply]
A darle bro

**Comment by Ricardo-g-garcia** [comment]
Hola, tengo la duda de como subir mi pagina a github pages, ya que tengo un repositorio que se llama frontend mission y dentro tengo tanto la carpeta de la practica de abogabot y la de la pastelería
entonces la pregunta es como subo a github pages solo la de la pastelería.

**Reply by hackit92** [reply]
Te recomiendo crear un nuevo repositorio exclusivamente para el proyecto de la pastelería y subir todos los archivos directamente a la raíz.

**Reply by JonathanJRodriguez** [reply]
La solución de hackit92  sería lo más correcto..
Pero igual podrías probar como lo sugiere ArielCanedo y habilitar pages para todo tu repositorio, y después en el readme pones un vínculo a cada una de las páginas que vayas haciendo.

**Comment by ArielCanedo** [comment]
Exacto, podrías poner el readme bonito e ir separando todo
Pero igual recomendaría varios repo

**Comment by LuceroLuciano** [comment]
Gracias por el dato explorer!

**Comment by LuceroLuciano** [comment]
Gracias por el dato explorer!
